app/src/main/java/org/openobservatory/ooniprobe/activity/MeasurementDetailActivity.java

```diff
@@ -211,14 +211,20 @@ private void load() {
 
     @Override
     public boolean onCreateOptionsMenu(Menu menu) {
-        if (measurement.hasLogFile(this))
-            getMenuInflater().inflate(R.menu.measurement, menu);
-        else
-            getMenuInflater().inflate(R.menu.measurement_nolog, menu);
-
+        getMenuInflater().inflate(R.menu.measurement, menu);
         return super.onCreateOptionsMenu(menu);
     }
 
+    @Override
+    public boolean onPrepareOptionsMenu(Menu menu) {
+        invalidateOptionsMenu();
+        if (measurement.hasLogFile(this))
```

Comment:
Are you sure this shan't be `!measurement.hasLogFile(this)`?

Reply:
right